cluster/rbac/manager.go

```diff
@@ -44,6 +44,10 @@ func (m *Manager) GetUsersForRole(role string) ([]string, error) {
 	return m.authZ.GetUsersForRole(role)
 }
 
+func (m *Manager) HasPermission(role string, permission *authorization.Policy) (bool, error) {
+	return m.authZ.HasPermission(role, permission)
+}
+
```

Comment:
**question:** do we wanna do it in a consistent way, like the we want to handle EC ? 

because with this approach it would be affected by EC and to overcome that we need to ask the leader

Reply:
raft consistency added by default